src/components/Layout/LayoutFooter.tsx

```diff
@@ -0,0 +1,30 @@
+import { forwardRef, ReactElement } from 'react'
+import { Slot } from '@spark-ui/slot'
+import { cx } from 'class-variance-authority'
+
+export interface LayoutFooterProps {
+  asChild?: boolean
+  children: ReactElement
+  className?: string
+}
+export const LayoutFooter = forwardRef<HTMLElement, LayoutFooterProps>(
+  ({ children, className }, forwardedRef) => (
+    <Slot
```

Comment:
The `Footer` component should be a `footer` tag IMO. We could also add the content here already because it's always going to be the same for all pages

Reply:
Layout Footer is an slot. Place whatever you want (normally footer tag, but it can also be wrapped inside other elements)

Reply:
Can you give an example ?

Reply:
```
<Layout.Footer>
  <div>
    <Banner />
    <Footer />
  <div>
<Layout.Footer>
```

Reply:
That behaviour ATM could be achieved using the `Container` component